### ISSUE TITLE
Integration with CM Answers Pro / Problem with controlling the visibility of notifications

### ISSUE THREAD

**Issue by MythThrazz** [buddyboss/buddyboss-platform#1319]
Hi,

We at CreativeMinds are developing a custom solution for one of our customers whose site is based on BuddyBoss. One of the requirements was that the CPT created with our plugin (Questions and Answers) should appear in the Activity screen. We've managed to find the right hooks to do that. Then client wanted to also add the activity notification about the group (for example Question added by/to the members of the group). We've managed to add a notification as well. But client also asked for an option to add Questions/Answers anonymously (by logged in users) and that's where problems began. 

First of all during testing we've found out that for the author of question added in the group, there are three notifications added about the single question. We've tried to check the options for controlling the Activity appearance, but the existing options are very limited, we also couldn't find a way to create our own options. 

**To sum up:**
- We integrate our plugin CM Answers Pro with BuddyPress
- We created the custom activity notifications for our own CPT
- We couldn't find/create the options allowing us to control where each custom activity (individual or group) notification appears.

**Comment by mehulkaklotar** [comment]
@MythThrazz In Platform, We are creating activity for CPT post and when someone replies to the activity post, we are creating notification for it. And if activity is sync with post then if someone replies to original post, it syncs comments to activity post.

To answer your question regarding having you own options in Platform settings, you can refer to this addon we have created where you find the code for having options page. https://github.com/buddyboss/buddyboss-platform-addon

You can change your activity post's appearance with overriding template or you can use in built hooks and filters to change it.

Let us know if you need anything.

**Comment by marienel21** [comment]
Hi @MythThrazz,

Thank you for your feature request.

We have imported this request into our product management system and will review it when planning new product features. You can see the features we're working on, as well as suggest new features, on our public roadmap: http://buddyboss.com/roadmap

I'll be closing this issue now in order to keep our GitHub issues focused on tasks that are ready for development.

Marienel
BuddyBoss Product Team
